File: practice_4-29/prac.py
def find_owed(items, index, money):
    shared_total = 0
    # for i in range(0, len(items)):
    #     if (i != index):
    #         shared_total += items[i]
    i = 0
    while i < len(items):
        if i != index:
            shared_total += items[i]
        i += 1
    owed = shared_total / 2
    print(str(money - owed))


def main():
    # items = [9.5, 11.75, 18.5, 3, 3]
    # index = 0
    # money = 30
    # items = [3, 4]
    # index = 1
    # money = 4
    # items = [1]
    # index = 0
    # money = 4
    items = [9.5, 11.75, 18.5, 3, 3]
    index = 2
    money = 0
    find_owed(items, index, money)


if __name__ == "__main__":
    main()

File: practice_4-29/Prac.java
public class Prac {
    public static void main(String[] args) {
        // double[] items = { 9.5, 11.75, 18.5, 3, 3 };
        // int index = 0;
        // double money = 30;
        // double[] items = { 3, 4 };
        // int index = 1;
        // double money = 4;
        // double[] items = { 1 };
        // int index = 0;
        // double money = 1;
        double[] items = { 3, 4 };
        int index = 1;
        double money = 0;
        findOwed(items, index, money);
    }

    public static void findOwed(double[] items, int index, double money) {
        double sharedTotal = 0;
        // for (int i = 0; i < items.length; i++) {
        // if (i != index) {
        // sharedTotal += items[i];
        // }
        // }
        int i = 0;
        while (i < items.length) {
            if (i != index) {
                sharedTotal += items[i];
            }
            i++;
        }
        double owed = sharedTotal / 2;
        System.out.println("Amount Back: " + (money - owed));
    }
}
